apps/sdk-vite-integration/src/components/TransferLogs.tsx

```
@@ -4,8 +4,8 @@ import {
     FilterTransferLogsOptions
 } from '@vechain/sdk-network';
 import { Link } from 'react-router-dom';
-import { Transfer } from "../types";
-import { explorerUrl, thorClient } from "../const";
+import { Transfer } from "../types/index.tsx";
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)